homeassistant/components/api.py

```diff
@@ -312,16 +315,30 @@ def post(self, request, domain, service):
         Returns a list of changed states.
         """
         hass = request.app['hass']
+        if not hass.services.has_service(domain, service):
```

Comment:
The service does not exist locally, however it could still exist in a multi-tentant setup. We don't really use that right now but changing that behavior is more than just changing the API.